src/main/webapp/WEB-INF/tags/instructor/course/courseEditCourseInfo.tag

```diff
@@ -0,0 +1,56 @@
+<%@ tag description="instructorCourseEdit - Course Info Panel" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ tag import="teammates.common.util.FieldValidator" %>
+<%@ attribute name="deleteCourseButton" type="teammates.ui.template.ElementTag" required="true" %>
+<%@ attribute name="course" type="teammates.common.datatransfer.CourseAttributes" required="true" %>
+<%@ attribute name="googleId" required="true" %>
```

Comment:
Is this equivalent to `${data.account.googleId}`?
It should be okay to use it like a constant if so.